### PR TITLE
Minor update for zh-CN translations

### PR DESCRIPTION
**Pull request by imgradeone**
Updated some strings for Chinese (Simplified) that are from recent updates on Geckium. Some strings are still untranslated due to uncertainty. Further study (on historical Chromium releases, I mean) on strings is still required for me.

Still seeking for help.

### CODE REVIEW

File: Profile Folder/chrome/locale/zh-CN/dtd/gsettings.dtd [hunk 1]
```diff
@@ -108,9 +108,9 @@
 <!ENTITY gsettings.appearance                           "外观">
 <!ENTITY gsettings.openChrThemesFolder				    "打开 Chromium 主题文件夹">
 <!ENTITY gsettings.refreshList					        "刷新列表">
-<!ENTITY gsettings.filename								"[!!! Fïℓèñá₥è: !!!]">
+<!ENTITY gsettings.filename								"文件名：">
 <!ENTITY gsettings.viewStorePage						"[!!! Vïèω §ƭôřè ƥáϱè !!!]">
```
Comment (angelbruni) [inline Profile Folder/chrome/locale/zh-CN/dtd/gsettings.dtd:112]: `viewStorePage` is the button that will take you to a theme's page.
Examples:
image
https://addons.mozilla.org/en-US/firefox/addon/chiaki-nanami-blue/
image
https://chromewebstore.google.com/detail/angry-birds/coaaadiiedpfjmpkampmbbfbnolpchik

File: Profile Folder/chrome/locale/zh-CN/properties/gsettings.properties [hunk 2]
```diff
@@ -1,12 +1,12 @@
-geckiumSettings=[!!! Gèçƙïú₥ §èƭƭïñϱƨ !!!]
+geckiumSettings=Geckium 设置
 
-addAnApp=Add an app
+addAnApp=添加应用程序
 
 themeHasNoDescription=This theme has no description.
-builtInTheme=[!!! [ßÛÌ£T-ÌN] {{themeName}} !!!]
+builtInTheme=[内置] {{themeName}}
 showInFolder=[!!! §λôω ïñ ƒôℓδèř !!!]
```
Comment (angelbruni) [inline Profile Folder/chrome/locale/zh-CN/properties/gsettings.properties:7]: `showInFolder` is the button that will take you to a theme's folder and selects it in the file manager.
image

**Comment by imgradeone** [comment]
Just fixed those strings in the recent commit.

(I'm still new to GitHub, and I hope that I didn't screw it up.)

**Review by angelbruni — APPROVED** [review]

**Comment by angelbruni** [comment]
Looks good, thank you!

**Pull request merged**